Sources/MockoloFramework/Parsers/SwiftSyntaxExtensions.swift

```diff
@@ -6,27 +6,23 @@
 //
 
 import Foundation
-#if canImport(SwiftSyntax)
 import SwiftSyntax
-#endif
-#if canImport(SwiftSyntaxParser)
-import SwiftSyntaxParser
-#endif
+import SwiftParser
 
-extension SyntaxParser {
-    public static func parse(_ fileData: Data, path: String) throws -> SourceFileSyntax {
-        // Avoid using `String(contentsOf:)` because it creates a wrapped NSString.
-        let source = fileData.withUnsafeBytes { buf in
-            return String(decoding: buf.bindMemory(to: UInt8.self), as: UTF8.self)
+extension Parser {
+    public static func parse(_ fileData: Data, path: String) -> SourceFileSyntax {
```

Comment:
Sorry to pick up tiny refactoring but shall we remove `path` parameter because it might not be used anywhere?
(Or, I think we can delete this method)

Reply:
OK. This function should be considered for backward compatibility because it is public, but this function may not be used from users. it can be deleted.
memo: impl commit https://github.com/uber/mockolo/commit/622227b0c5df0fcdc9913139bbdd2ef5f1632aeb